app/assets/stylesheets/print.scss

```diff
@@ -0,0 +1,24 @@
+@media print {
+  // header
```

Comment:
maybe we can avoid global rules here and only apply them in the statement pages, as worried someone else wants to print something on another page fully and we mess with that here? let me know your thoughts

Reply:
I talked to Mark and Shahad about this and they agree that would be the ideal approach. Its a bit tricky as some of the CSS selectors are global (header/footer) so I've split it out into a new top-level CSS file that gets compiled and included manually by views that want to use it.